docs/build/_category_.json

```
@@ -2,7 +2,9 @@
   "label": "Build",
   "position": 3,
   "link": {
-    "type": "doc",
-    "id": "build/quick-start/index"
+    "type": "generated-index",
+    "title": "Build with Yellow Network",
```

```suggestion
    "title": "Build with Yellow SDK",
```

App devs build **on** the the Yellow Network **with** the Yellow SDK